backend/controller/controller.go

```diff
@@ -138,12 +140,18 @@ type clients struct {
 	runner ftlv1connect.RunnerServiceClient
 }
 
+type ControllerListListener interface {
```

Comment:
Rather than scheduledtask and cronjob services both polling the db for the controller list to update their hashrings, I've centralised the polling into the controller, which then broadcasts it to listeners